shared-components/statviz/components/visualizations/demographic/DemographicPyramid.tsx

```diff
@@ -45,28 +45,100 @@ export default function DemographicPyramid({
 }: IDemographicChartProps) {
   const onExport = getOnExport(BarChartCenterAxis);
 
-  const prepareFacts = () => {
+  const prepareFactsForGraph = () => {
     const dataXr = tidy(
       demographics?.facts as BeneficiaryDemographicsResult[],
-      filter((value) => value.gender === "Male"),
+      filter((value) => value.gender === "Male" && value.age !== null),
       groupBy("age", [summarize({ count: sum("count") })]),
       map((value) => ({ x: value.count, y: value.age ?? 0 })),
```

Comment:
I suppose the `?? 0` can be removed since null age is filtered out

Reply:
seems that it is still needed for typecasting. value.age according to its type could also be undefined. So, we need the ?? still that the type changes from number | undefined to number

Reply:
I replaced now with the ! operator.